### PR TITLE
Tweaks pill bottle sprites

### LINKED ISSUE #8604: Pillbottle colo(u)red labels need to be more visible
How they are right now:
mspaint_2016-03-02_23-25-29

How they should be (note I am bad spriteperson):
mspaint_2016-03-02_23-28-18

Or alternatively, color the whole bottle:
mspaint_2016-03-02_23-25-292

### PR DESCRIPTION
**Pull request by Intigracy**
Red / Yellow / Blue / White/Default
Before:

After: (Currently in PR)

blue / darkblue / green / orange / purple / red / yellow / grey / white/default

Any other ideas to do instead?

I could try the whole pill bottle.

closes #8604

Also alt clicking bottles to change the label instead of just right click.

### CODE REVIEW

**Comment by Probe1** [comment]
I can't pick.

**Comment by Probe1** [comment]
how about none of the above.  the orange bottle and white cap is a ubiquitous pill bottle look that everyone will instantly recognize.  there's no need to mess with that.  

however the black label does offer much better contrast for the colors.

(hell why not have both current and black label in game, variety doesn't hurt.  why not all of the above)

**Comment by 9600bauds** [comment]
I really don't like how it looks by having the pillbottle white and coloring the cap instead
Re-posting my mockups from the original feature request:
mspaint_2016-03-02_23-28-18
mspaint_2016-03-02_23-25-292

**Comment by Probe1** [comment]
shit I like those too, just throw in a lot of pill bottle variations where's the drawback?

**Comment by 9600bauds** [comment]
Probably tedious to code and/or tedious to pick while in-game

**Comment by ghost** [comment]
Make alt click bring up the colour menu please

**Comment by Intigracy** [comment]
@DrCelt done

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -156,6 +156,12 @@
 			return
 	return
 
+/obj/item/weapon/storage/pill_bottle/AltClick()
+	if(!usr.isUnconscious() && Adjacent(usr))
```
Comment (ghost) [inline code/game/objects/items/weapons/storage/firstaid.dm:160]: Have you tested this? It's a verb you're calling, not a proc

**Comment by ghost** [comment]
@intigracy 👌, but can you do baud's one here, it looks far far nicer on the eyes. https://cloud.githubusercontent.com/assets/6526157/13482993/4d1cd04c-e0cf-11e5-8da4-d9176135755b.png

**Comment by Probe1** [comment]
I was against that one because the white cap and orange container is the iconic look of a pill bottle and makes it immediately identifiable to anyone.

**Comment by 9600bauds** [comment]
Idunno, I don't think there are a lot of small rectangular shapes of specific colors with a label and cap in Medbay to get mixed up between
In any case it'd be nice as an option

File: code/game/objects/items/weapons/storage/firstaid.dm (hunk 1 above)
Comment (Intigracy) [inline code/game/objects/items/weapons/storage/firstaid.dm:160]: Yes, I tested it when I took the picture with all the colors in it..
Verbs are just special procs.

Comment (PJB3005) [inline code/game/objects/items/weapons/storage/firstaid.dm:160]: @intigracy in fact verbs are only different to procs in that they automatically get added to the `verbs` list.

**Comment by Shadowmech88** [comment]
I personally don't have any problem with the existing sprites.

**Comment by 9600bauds** [comment]
This is for the optional color labels by the way, does not affect default pillbottles

**Comment by clusterfack** [comment]
At first I wasn't keen but the sprites improved

**Pull request merged**